src/VideoIntelligence/README.md

```diff
@@ -1,6 +1,6 @@
 # Google Cloud PHP Video Intelligence
 
-> Idiomatic PHP client for [Cloud Video Intelligence](https://cloud.google.com/video-intelligence/).
```

Comment:
You accidentally removed the tag line on this one.

Reply:
I removed it on purpose, as it's not present in the README for any other GAPIC-only client.